deploy/examples/operator.yaml

```diff
@@ -101,7 +101,7 @@ data:
   # The default version of CSI supported by Rook will be started. To change the version
   # of the CSI driver to something other than what is officially supported, change
   # these images to the desired release of the CSI driver.
-  # ROOK_CSI_CEPH_IMAGE: "quay.io/cephcsi/cephcsi:v3.6.2"
+  # ROOK_CSI_CEPH_IMAGE: "quay.io/cephcsi/cephcsi:v3.7.0"
```

Comment:
@Madhu-1 The min version has changed, but this line is about the default version, no?

Reply:
#10759 updated the default version to 3.7, but this update had been missed

Reply:
This is always the default value as Travis mentioned it got missed in last update